tests/test_smile.py

```diff
@@ -849,35 +829,34 @@ async def test_connect_anna_without_boiler_fw4(self):
     # Actual test for directory 'Adam'
     # living room floor radiator valve and separate zone thermostat
     # an three rooms with conventional radiators
-    """
-    @pytest.mark.asyncio
-    async def test_connect_adam(self):
-        testdata = {
-            "95395fb15c814a1f8bba88363e4a5833": { "temperature": 19.8, 'active_preset': 'home',},
-            "450d49ef2e8942f78c1242cdd8dfecd0": { "temperature": 20.18, 'battery':  0.77, 'selected_schedule': 'Kira' },
-            "bc9e18756ad04c3f9f35298cbe537c8e": { "temperature": 20.63, 'thermostat': 20.0 },
-        }
 
-        self.smile_setup = 'adam_living_floor_plus_3_rooms'
-        server, smile, client = await self.connect_wrapper()
-        assert smile.smile_type == "thermostat"
-        assert smile.smile_version[0] == "2.3.35"
-        assert not smile._smile_legacy
-        await self.device_test(smile, testdata)
-        await self.tinker_thermostat(
-            smile, "95395fb15c814a1f8bba88363e4a5833", good_schemas=["Living room"]
-        )
-        await smile.close_connection()
-        await self.disconnect(server, client)
-
-        server, smile, client = await self.connect_wrapper(put_timeout=True)
-        await self.tinker_thermostat(
-            smile, "95395fb15c814a1f8bba88363e4a5833", good_schemas=["Living room"],
-            unhappy=True,
-        )
-        await smile.close_connection()
-        await self.disconnect(server, client)
-    """
+    # @pytest.mark.asyncio
+    # async def test_connect_adam(self):
+    #    testdata = {
+    #        "95395fb15c814a1f8bba88363e4a5833": { "temperature": 19.8, 'active_preset': 'home',},
+    #        "450d49ef2e8942f78c1242cdd8dfecd0": { "temperature": 20.18, 'battery':  0.77, 'selected_schedule': 'Kira' },
+    #        "bc9e18756ad04c3f9f35298cbe537c8e": { "temperature": 20.63, 'thermostat': 20.0 },
+    #    }
+
+    #    self.smile_setup = 'adam_living_floor_plus_3_rooms'
+    #    server, smile, client = await self.connect_wrapper()
+    #    assert smile.smile_type == "thermostat"
+    #    assert smile.smile_version[0] == "2.3.35"
+    #    assert not smile._smile_legacy
+    #    await self.device_test(smile, testdata)
+    #    await self.tinker_thermostat(
+    #        smile, "95395fb15c814a1f8bba88363e4a5833", good_schemas=["Living room"]
+    #    )
+    #    await smile.close_connection()
+    #    await self.disconnect(server, client)
+
+    #    server, smile, client = await self.connect_wrapper(put_timeout=True)
+    #    await self.tinker_thermostat(
+    #        smile, "95395fb15c814a1f8bba88363e4a5833", good_schemas=["Living room"],
+    #        unhappy=True,
+    #    )
+    #    await smile.close_connection()
+    #    await self.disconnect(server, client)
```

Comment:
Shouldn't we remove this uncommented code?

Reply:
Yes, we probably should. @CoMPaTech this is mostly your code, you decide :)

Reply:
I think I already confirmed so in an earlier review - but - that we should remove the accompanying user-data as well (otherwise we really won't know which data is which) :)

Reply:
OK, I'll have a look.

Reply:
@CoMPaTech Can I also remove the #TODO?

Reply:
Which one?

```
plugwise/messages/responses.py:        # TODO: unused crc
plugwise/messages/responses.py:    TODO:
plugwise/messages/responses.py:    TODO:
plugwise/messages/responses.py:    - 4 : TODO: Unknown
plugwise/messages/responses.py:    TODO: FeatureBitmask
Binary file plugwise/messages/__pycache__/requests.cpython-38.pyc matches
Binary file plugwise/messages/__pycache__/responses.cpython-38.pyc matches
plugwise/messages/requests.py:    """TODO: PublicNetworkInfoRequest
plugwise/messages/requests.py:        # TODO: Make sure that '01' means enable, and '00' disable joining
plugwise/messages/requests.py:    TODO: Some kind of reset request
plugwise/messages/requests.py:        TODO: PWEswImagePrepareRequestV1_0
plugwise/messages/requests.py:        Response message: TODO:
plugwise/messages/requests.py:    Response message: TODO:
plugwise/messages/requests.py:    Response message: TODO:
plugwise/messages/requests.py:    Response message: TODO:
plugwise/messages/requests.py:    Response message: TODO:
plugwise/messages/requests.py:    Response message: TODO:
plugwise/messages/requests.py:    TODO:
plugwise/messages/requests.py:    Response message: TODO:
plugwise/stick.py:        # TODO: flake8 indicates scan is too complex, level 23 indenting is indeed complex
plugwise/stick.py:        # TODO: flake8 indicates scan is too complex, level 47! indenting is indeed overly complex
plugwise/stick.py:        # TODO: flake8 indicates scan is too complex, level 34! indenting is indeed too complex
plugwise/stick.py:        # TODO: flake8 indicates scan is too complex, level 28 indenting is indeed complex
plugwise/nodes/sed.py:# TODO:
plugwise/nodes/scan.py:        # TODO:
plugwise/constants.py:NACK_SLEEP_SET = b"00F7"  # TODO: Validate
plugwise/constants.py:SED_AWAKE_UNKNOWN = 4  # TODO: Unknown
plugwise/constants.py:# TODO: Need to validate RSSI sensors
plugwise/message.py:    # TODO: none of the self objects are predefined
plugwise/smile.py:        # TODO create this as another function NOT part of connect!
plugwise/smile.py:        # TODO: add locations with members as appliance as well
plugwise/parser.py:        # TODO: flake8 indicates scan is too complex, level 35 indenting is indeed too complex
tests/test_smile.py:    # TODO: This device setup needs work - doesn't seem to work straightforard
tests/INIT.md:Todo: (** ensure you are in your venv**)
```

Reply:
```
    # TODO: This device setup needs work - doesn't seem to work straightforard
    # currently breaks on setting thermostat setpoint
```

Reply:
yeah sure, that can go as well ...